bin/check_ldap.rb

```diff
@@ -0,0 +1,21 @@
+# This is a rails runner that will print the status of the database
+# Possible outcomes:
+#
+#   * `LDAP_READY`: connection successfull
+#   * `LDAP_FAIL`: cannot authenticate within ldap server
+#   * `LDAP_DOWN`: cannot connect to the ldap server
+
+def ldap_ready?
+  ldap_config = Velum::LDAP.ldap_config
+  ldap = Net::LDAP.new(ldap_config)
+
+  if ldap.bind
```

Comment:
I also thought about simplifying this to just 

```
ldap.bind
puts "LDAP_READY"
```

because what we want to know is if ldap instance is up or not. It doesn't matter if it's able to authenticate or not. If we do that change we can change the `*` in the bash script to `LDAP_READY`.